R-package/R/lightgbm.R

```diff
@@ -44,7 +44,7 @@
 #'                 }
 #'             }
 #' @param eval_freq evaluation output frequency, only effective when verbose > 0 and \code{valids} has been provided
-#' @param init_model path of model file of \code{lgb.Booster} object, will continue training from this model
+#' @param init_model path or model file of \code{lgb.Booster} object, will continue training from this model
```

Comment:
```suggestion
#' @param init_model path of model file or \code{lgb.Booster} object, will continue training from this model
```

Reply:
ha didn't notice this! Yes, agree with @jmoralez , please make this change.